kuma/dashboards/jinja2/dashboards/spam.html

```diff
@@ -0,0 +1,95 @@
+{% extends 'base.html' %}
+{% set styles = ('dashboards',) %}
+{% set title = 'Spam Dashboard' %}
+
+{% block title %}{{ page_title(title) }}{% endblock %}
+
+{% set stat_ids=(('spam_viewers_change', '% Change', 'rate'),
+                 ('spam_viewers', 'Spam Viewers', 'data'),
+                 ('spam_viewers_daily_average', 'Daily Average Viewers', 'float'),
+                 ('published_spam', 'Published Spam', 'data'),
+                 ('blocked_spam', 'Blocked Spam', 'data'),
+                 ('blocked_ham', 'Blocked Ham', 'data'),
+                 ('true_positive_rate', 'True Positive Rate', 'rate'),
+                 ('true_negative_rate', 'True Negative Rate', 'rate')) %}
+{% block content %}
+<h1>{{ title }}</h1>
+{% if improperly_configured %}
+  <p>{{ improperly_configured }}</p>
```

Comment:
@stephaniehobson what format should be used to turn this into an error?  The content is something like:

```
<p>GOOGLE_ANALYTICS_CREDENTIALS Constance setting is badly formed.</p>
```